pkgs/applications/audio/faust/faust2.nix

```diff
@@ -194,8 +196,8 @@ let
         # export parts of the build environment
         for script in "$out"/bin/*; do
           wrapProgram "$script" \
-            --set FAUSTLIB "${faust}/lib/faust" \
-            --set FAUST_LIB_PATH "${faust}/lib/faust" \
+            --set FAUSTLIB "${faust}/share/faust" \
```

Comment:
I asked, and the answer was [yes](https://github.com/grame-cncm/faust/issues/54).